vars/common-vars.yml

```diff
@@ -31,7 +31,18 @@ socok8s_repos_to_configure:
   openSUSE-Leap-15.0-Update: "{{ obs_mirror }}/update/leap/15.0/oss/"
   openSUSE-Leap-15.0-Cloud: "{{ obs_mirror }}/repositories/Cloud:/Tools/openSUSE_Leap_15.0/"
   Leap15develtools: "{{ obs_mirror }}/repositories/devel:/tools/openSUSE_Leap_15.0/"
+  SLE-15-SP1-JeOS-GM-Full: http://download.suse.de/full/full-sle15-sp1-x86_64/
```

Comment:
we really should use download.suse.de or download.o.org in the socok8s jobs but use the provo local mirrors (as indicated by {{ obs_mirror }} for example

Reply:
AFAIK this was really a last resort when we were not able to find out all the right repositories for SP1 packages